packages/backend/src/rafiki/graphql.client.ts

```diff
@@ -0,0 +1,14 @@
+import {request} from 'graphql-request'
+import env from '../config/env'
+import {Mutation, Query} from "./generated/graphql";
+
+const GRAPHQL_ENDPOINT = env.GRAPHQL_ENDPOINT
+
+export async function requestGQL<T extends Partial<Mutation | Query>>(query: string, variables?: Record<string, unknown>, headers?: Record<string, string>): Promise<T> {
```

Comment:
Should we have a default for `headers` that is set to an empty object? This way we will not have to pass an empty object as the 3rd arg when using `requestGQL`.

Reply:
removed empty objects headers